src/style/stars.css

```
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(/src/images/star.svg);
```

not critical but you can combine it
```suggestion
  margin-right: 4px;
  width: 16px;
  height: 16px;
  background: url(/src/images/star.svg) center no-repeat;
```